src/ice_match/utils/conversion_helpers.py

```
@@ -85,17 +85,17 @@ def convert_mt_to_bbl_with_product_ratio(
     """
     product_ratio = get_product_conversion_ratio(product_name, config_manager)
     converted_bbl = quantity_mt * product_ratio
-    
+
     logger.debug(
         f"MT→BBL conversion: {quantity_mt} MT × {product_ratio} = {converted_bbl} BBL for {product_name}"
```

Ruff (0.12.2)

String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)